world/test_combat.py

```diff
@@ -494,6 +496,11 @@ def test_do_nothing(self):
 @patch('world.rulebook.utils.delay', new=CALL_IMMEDIATE)
 class AinneveCombatAttackTestCase(AinneveCombatTest):
     """Tests for attack combat actions in Ainneve."""
+    def setUp(self):
+        super(AinneveCombatAttackTestCase, self).setUp()
+        settings.PROTOTYPE_MODULES = ainneve_settings.PROTOTYPE_MODULES
```

Comment:
This can likely be removed as a workaround from the era before `evennia test --settings settings.py gamedir` was supported.

Reply:
Ah, no, actually this is needed. It is certainly possible that we shouldn't have to, but as noted below, on 0.7 we'll get a spawner error. Probably worth a bug report to Evennia core though.